src/components/authorization/policies/by-role/project-manager.policy.ts

```diff
@@ -1,57 +1,59 @@
-import { member, Policy, Role, sensMediumOrLower } from '../util';
+import { inherit, member, Policy, Role, sensMediumOrLower } from '../util';
 
 // NOTE: There could be other permissions for this role from other policies
-@Policy(Role.ProjectManager, (r) => [
-  r.Education.create,
-  r.Producible.create,
-  r.Project.create,
-  r.Engagement.create,
-  r.Partnership.create,
-  r.Unavailability.create,
-  r.User.create,
-  Role.assignable(r, [
-    Role.Intern,
-    Role.Liaison,
-    Role.Mentor,
-    Role.RegionalCommunicationsCoordinator,
-    Role.Translator,
-  ]),
+@Policy(
+  [Role.ProjectManager, Role.RegionalDirector, Role.FieldOperationsDirector],
+  (r) => [
+    Role.assignable(r, [
+      Role.Intern,
+      Role.Liaison,
+      Role.Mentor,
+      Role.RegionalCommunicationsCoordinator,
+      Role.Translator,
+    ]),
 
-  r.Budget.edit,
-  r.BudgetRecord.edit,
-  r.Ceremony.edit,
-  r.FileNode.edit,
-  r.Education.read,
-  r.Producible.edit,
-  r.EthnologueLanguage.read,
-  r.FundingAccount.read,
-  r.Language.read,
-  r.Engagement.edit.specifically((p) => p.disbursementCompleteDate.read),
-  r.Organization.read,
-  r.Partner.read
-    .specifically((p) => p.pmcEntityCode.none)
-    .children((c) => c.posts.edit),
-  r.Partnership.read
-    .specifically((p) => [
-      p.many('agreement', 'agreementStatus', 'types', 'organization', 'primary')
-        .edit,
-      p.partner.whenAny(member, sensMediumOrLower).edit,
-    ])
-    .when(member).create,
-  r.Post.edit,
-  r.Product.edit.create,
-  r.Project.edit.specifically((p) => [
-    p
-      .many('rootDirectory', 'otherLocations', 'primaryLocation')
-      .whenAny(member, sensMediumOrLower).edit,
-  ]),
-  r.FieldRegion.edit,
-  r.FieldZone.edit,
-  r.ProjectMember.edit.create,
-  r.PeriodicReport.read.when(member).edit,
-  r.User.read,
-  r.Unavailability.read,
-  r.ProjectChangeRequest.edit.when(member).create,
-  r.StepProgress.read.when(member).edit,
-])
+    r.Budget.read.when(member).edit,
+    r.BudgetRecord.read.when(member).edit,
+    r.Ceremony.read.when(member).edit,
+    r.Education.read.create,
+    ...inherit(
+      r.Engagement.read.create
+        .when(member)
+        .edit.specifically((p) => p.disbursementCompleteDate.read),
+      r.LanguageEngagement.specifically((p) => [
+        p.paratextRegistryId.when(member).read,
+      ])
+    ),
+    r.EthnologueLanguage.read,
+    r.FieldRegion.read,
+    r.FieldZone.read,
+    r.FundingAccount.read,
+    r.Language.read,
+    r.Organization.read,
+    r.Partner.read
+      .specifically((p) => p.pmcEntityCode.none)
+      .children((c) => c.posts.read.create),
+    r.Partnership.whenAny(member, sensMediumOrLower).read.create.specifically(
+      (p) => [
+        p.many('agreement', 'agreementStatus', 'types', 'partner', 'primary')
+          .edit,
+      ]
+    ),
+    r.PeriodicReport.read.when(member).edit,
+    r.Producible.when(member).edit.create,
```

Comment:
```suggestion
    r.Producible.edit.create,
```
Producibles are global objects. They are only related via `Product.produces`.